src/rttr/type.cpp

```diff
@@ -309,8 +315,11 @@ method type::get_method(string_view name) const RTTR_NOEXCEPT
 method type::get_method(string_view name, const std::vector<type>& type_list) const RTTR_NOEXCEPT
 {
     const auto raw_t = get_raw_type();
-    for (const auto& meth : raw_t.m_type_data->get_class_data().m_methods)
+
+    const auto& methvec = raw_t.m_type_data->get_class_data().m_methods;
+    for (auto mit = methvec.crbegin() ; mit != methvec.crend() ; mit++)
```

Comment:
better use prefix operator:
`for (auto mit = methvec.crbegin() ; mit != methvec.crend() ; ++mit)`

Reply:
done